kepub/converter.go

```diff
@@ -119,7 +120,15 @@ func (c *Converter) GetHtmlFiles(dir string) ([]string, error) {
 			}
 		}
 		if (mimeType == "text/html" || mimeType == "application/xhtml+xml") && href != "" {
-			files = append(files, filepath.Join(filepath.Dir(opfPath), href))
+			//Most of the time colon means href is a remote address.
+			if !strings.Contains(href, ":") {
+				file := filepath.Join(filepath.Dir(opfPath), href)
```

Comment:
This logic is incorrect. Instead, do this:

1. Split by `#`, get `[0]` ([why](https://github.com/w3c/epubcheck/blob/794b7ce290c32ea8c3f4a5e4f441aac11fcf2484/src/main/java/com/adobe/epubcheck/ctc/epubpackage/EpubPackage.java#L167-L197)).
2. Use `filepath.FromSlash` before processing the path (package documents use forward slashes).

Note that you don't need to check if it's a URL because of [this restriction](https://www.w3.org/publishing/epub3/epub-spec.html#sec-foreign-restrictions).